### ISSUE TITLE
Attaching to ClearScript with Node.js Tools for Visual Studio 2017

### ISSUE THREAD

**Issue by ezolotko** [Microsoft/ClearScript#43]
Hi,

I tried to attach Node.js Tools for Visual Studio 2017 debugger to a running ClearScript application, but couldn't get it to work.
Here is how I initialize ClearScript:
`new V8ScriptEngine("Name", V8ScriptEngineFlags.EnableDebugging | V8ScriptEngineFlags.EnableRemoteDebugging, 4545))
`
The script I am running is basically an endless loop:
```
System.Console.WriteLine("Hello");

i = 0;
while (true)
{
    System.Console.WriteLine(i);
    System.Console.WriteLine(i);
    System.Console.WriteLine(i);

    if (i % 2 == 0)
        System.Console.WriteLine("e");
    else
        System.Console.WriteLine("odd");

    i++;
}
```
Then, I try to attach from VS2017 (as described [here](https://stackoverflow.com/questions/46500302/attach-visual-studio-debugger-to-electron-app) and [here](https://github.com/Microsoft/nodejstools/wiki/Advanced-Debugging)).

When I try to connect using TCP ("Node.js remote debugging"), the dialog hangs after clicking the Find button, and gives the following error when I stop the script:
![image](https://user-images.githubusercontent.com/993271/35426057-b13220be-0292-11e8-8602-2526850a3264.png)

When I try to connect using Websockets ("WebKit websocket (no authentication)"), it does find the target, and gives me two "code types" to choose from:
![image](https://user-images.githubusercontent.com/993271/35426167-81b4ac48-0293-11e8-9823-dd7eee1f61d2.png)

When I choose "JavaScript (Node.js 8+)", it looks like it attaches to the server, but when I select "Break All" from Debug menu, it does nothing.

When I choose "WebKit" code type, I get the following error:
![image](https://user-images.githubusercontent.com/993271/35426048-a987be6e-0292-11e8-8070-976c8da78f23.png)

I tried plain ws:// URL without the GUID also. Could you please point me to the right direction?

**Comment by ClearScriptLib** [comment]
Hi @ezolotko,

Thanks for your report.

Using Visual Studio's "WebKit websocket" connection type, we were able to attach to a ClearScript V8 runtime and inspect variables in the Watch window, but much of the debugger was nonfunctional.

We then attached to a Node.js process running the same JavaScript test code (replacing `System.Console.WriteLine` with `console.log`), and observed exactly the same behavior.

We therefore conclude that the problem is external to ClearScript and recommend that you use Visual Studio Code or Chrome Debugging Tools.

Thanks!
